src/main/java/com/passpnu/passwordmanager/entity/Password.java

```diff
@@ -0,0 +1,26 @@
+package com.passpnu.passwordmanager.entity;
+
+import jakarta.persistence.*;
```

Comment:
Use single imports instead of wildcard ones for readability

Reply:
Corrected